recipes/gvcf-regions/build.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 mkdir -p $PREFIX/bin
-sed -i '1i#!/opt/anaconda1anaconda2anaconda3/bin/python' gvcf_regions.py
+sed -i $'1i\\\n#!/opt/anaconda1anaconda2anaconda3/bin/python\n' gvcf_regions.py
```

Comment:
Thanks so much for the reminder on this. I pushed a new version of the recipe which should, fingers crossed, support OSX correctly now with sed. Thanks again.